torch/csrc/jit/backends/nnapi/nnapi_backend_lib.cpp

```diff
@@ -0,0 +1,49 @@
+#include <torch/csrc/jit/backends/backend.h>
+#include <torch/csrc/jit/backends/backend_exception.h>
+
+namespace torch {
+namespace jit {
+
+// This file has no implementation yet, but the declarations are necessary to
+// register the backend properly and test preprocess
+// TODO T91991928: implement compile() and execute()
+class NnapiBackend : public PyTorchBackendInterface {
```

Comment:
Something to address next PR: existing code (mobile/contrib/nnapi/nnapi.h) with different writes "NNApi" rather than "Nnapi" as you did here. I think your way is more principled, but we should be make it consistent (I'd say we refactor the older code with the new form -- @iseeyuan what do you think?).

Reply:
I agree to unify them to "Nnapi". I saw "NN", "API", "NNAPI", "nnapi", etc., it would make sense to unify the format in code.